SConstruct

```diff
@@ -415,6 +415,27 @@ if windows:
 env = Environment(**envDict)
```

Comment:
Rather than loading some of the tools here, when the envDict is fed to Environment, and then some later, maybe just adorn the envDict's 'tools' field based on decide_platform_toolchain, and then load all of the tools at once?